context/app/static/js/components/detailPage/ProcessedData/ProcessedDataset/ProcessedDataset.tsx

```diff
@@ -65,12 +73,14 @@ function SummaryAccordion() {
   const [dateLabel, dateValue] = getDateLabelAndValue(dataset);
   return (
     <Subsection title="Summary" icon={<SummarizeRounded />}>
-      <ProcessedDatasetDescription />
-      <LabelledSectionText label="Consortium">{dataset.group_name}</LabelledSectionText>
-      <Contact />
-      <LabelledSectionText label={dateLabel}>
-        {dateValue ? formatDate(new Date(dateValue), 'yyyy-MM-dd') : 'N/A'}
-      </LabelledSectionText>
+      <Stack spacing={1}>
+        <ProcessedDatasetDescription />
+        <LabelledSectionText label="Consortium">{dataset.group_name}</LabelledSectionText>
```

Comment:
Is 'Consortium' the right label for group name? Should `group_name` be `mapped_consortium`? Do we also want to display the `group_name`? @tsliaw

Reply:
Consortium should be the label for `mapped_consortium`. I didn't include the group field initially since I assumed that the group should be the same for both the raw and processed, but that's probably not true. @austenem Let's add the group field before the consortium field in the summary section. Can you also replace the consortium field in the mini processed data detail section (the pop-up thing on the right) with group instead? That field should be more useful than consortium.

Reply:
@tsliaw Will do! Would it make sense to also replace the "Consortium" field in the summary view top bar, or to add a "Group" field?